src/components/layout/footer.tsx

```
@@ -0,0 +1,74 @@
+import { createResource } from 'solid-js'
+
+type Commit = {
```

ESLint

[error] 'Commit' is defined but never used.

(@typescript-eslint/no-unused-vars)